src/OrganisationRegistry.Api/Backoffice/Organisation/OrganisationKboController.cs

```diff
@@ -89,7 +89,7 @@ public async Task<IActionResult> TerminateKboCoupling([FromRoute] Guid id)
 
         /// <summary>Gets the termination status of an organisation coupled with the KBO.</summary>
         [HttpGet("{id}/kbo/{kboNumber}/termination")]
-        [OrganisationRegistryAuthorize(Roles = Roles.OrganisationRegistryBeheerder + "," + Roles.Developer)]
+        [OrganisationRegistryAuthorize(Roles = Roles.OrganisationRegistryBeheerder + "," + Roles.Developer + "," + Roles.VlimpersBeheerder)]
```

Comment:
Might want to further check if org is vlimpers org

Reply:
is het mogelijk om bovenstaande als volgt te doen ?
[OrganisationRegistryAuthorize(Roles.OrganisationRegistryBeheerder)]
[OrganisationRegistryAuthorize(Roles.Developer )]
[OrganisationRegistryAuthorize(Roles.VlimpersBeheerder)]

dan zit ge niet met die awkward string concatenation ...